pypdf/_encryption.py

```diff
@@ -941,10 +942,10 @@ def _make_crypt_filter(self, idnum: int, generation: int) -> CryptFilter:
     def _get_crypt(
         method: str, rc4_key: bytes, aes128_key: bytes, aes256_key: bytes
     ) -> CryptBase:
-        if method == "/AESV3":
```

Comment:
I suggest to not use `elif` here, but plain `if` statements. IMHO it is bad code style to have `elif` when every branch returns directly anyway.

Reply:
Amended: removed the `elif`.